frontend/src/pages/groups/GroupEditPage.tsx

```diff
@@ -16,15 +20,17 @@ const GroupEditPage: FunctionComponent = () => {
   })
 
   // Set up the mutation to update the group
-  // const [updateGroup, { loading: mutationIsLoading }] = useMutation(
-  //   UPDATE_GROUP,
-  // )
+  const [
+    updateGroup,
+    { loading: mutationIsLoading, error: mutationError },
+  ] = useUpdateGroupMutation()
 
   const onSubmit = (input: GroupUpdateInput) => {
-    // TODO support an updateGroup resolver on the backend
-    // updateGroup({ variables: { input } }).catch((error) => {
-    //   console.log(error)
-    // })
+    updateGroup({ variables: { id: parseInt(groupId, 10), input } }).catch(
+      (error) => {
+        console.error(error)
+      },
```

Comment:
Suggestion (non-blocking): no need for the wrapping lambda?

```typescript
.catch(console.error)
```

Reply:
Oh yeah good... catch